Flaky ScaleResiliencyTest$ScaleUp.shouldContinueScaleAfterRestart

**Summary**

```
Condition with alias 'until cluster topology is complete' didn't complete within 10 seconds because assertion condition with alias until cluster topology is complete 
Expecting all elements of:
  [TestStandaloneGateway{nodeId = gateway-0}]
to satisfy given requirements, but these elements did not:

TestStandaloneGateway{nodeId = gateway-0}
error: Expected replication factor to be <2> but was <1>.
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:985)
	at org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:769)
	at io.camunda.zeebe.qa.util.cluster.TestCluster.awaitCompleteTopology(TestCluster.java:306)
	at io.camunda.zeebe.it.clustering.dynamic.ScaleResiliencyTest$ScaleUp.shouldContinueScaleAfterRestart(ScaleResiliencyTest.java:141)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: java.lang.AssertionError: 
```

--- a/qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/dynamic/ScaleResiliencyTest.java
+++ b/qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/dynamic/ScaleResiliencyTest.java
@@ -113,7 +113,10 @@ class ScaleResiliencyTest {
       final var newClusterSize = INITIAL_CLUSTER_SIZE + 1;
       final var newBroker =
           createNewBroker(
-              newClusterSize, newClusterSize - 1, getDataDirectory(tmpDir, newClusterSize - 1));
+              newClusterSize,
+              newClusterSize - 1,
+              REPLICATION_FACTOR,
+              getDataDirectory(tmpDir, newClusterSize - 1));
 
       // scale to 3
       Utils.scale(cluster, newClusterSize);
@@ -143,7 +146,10 @@ class ScaleResiliencyTest {
     }
 
     private TestStandaloneBroker createNewBroker(
-        final int newClusterSize, final int newBrokerId, final Path dataDirectory) {
+        final int newClusterSize,
+        final int newBrokerId,
+        final int replicationFactor,
+        final Path dataDirectory) {
       final var newBroker =
           new TestStandaloneBroker()
               .withBrokerConfig(
@@ -151,6 +157,7 @@ class ScaleResiliencyTest {
                     b.getExperimental().getFeatures().setEnableDynamicClusterTopology(true);
                     b.getCluster().setClusterSize(newClusterSize);
                     b.getCluster().setNodeId(newBrokerId);
+                    b.getCluster().setReplicationFactor(replicationFactor);
                     b.getCluster()
                         .setInitialContactPoints(
                             List.of(

